bucket/unikey.json

```diff
@@ -0,0 +1,34 @@
+{
+    "version": "43RC5-200929",
+    "description": "UniKey is the most popular Vietnamese keyboard for Windows. The core UniKey Vietnamese Input Method is also the engine for many Vietnamese keyboards in Linux, macOS and iOS. UniKey is free of charge. Source code for the UniKey Vietnamese Input Method is distributed under the GNU General Public License.",
+    "homepage": "https://www.unikey.org",
```

Comment:
Use `https://www.unikey.org/en` instead. So it is friendlier to people who don't understand Vietnamese.